client/src/features/ProjectPageV2/utils/roleUtils.ts

```diff
@@ -51,9 +52,22 @@ export function toSortedMembers(members: ProjectMemberListResponse) {
     if (a.role !== b.role) {
       return toNumericRole(b.role) - toNumericRole(a.role);
     }
-    if (a.email && b.email) {
-      return a.email.localeCompare(b.email);
+    if (a.first_name && b.first_name) {
+      return a.first_name.localeCompare(b.first_name);
     }
     return a.id < b.id ? -1 : 1;
   });
 }
+
+export function getMemberNameToDisplay(
+  member: ProjectMemberResponse,
+  includeNamespace?: boolean
+): string {
+  const namespace =
+    includeNamespace && member.namespace ? ` @${member.namespace}` : "";
+  return member.first_name || member.last_name
+    ? `${member.first_name} ${member.last_name}${namespace}`
+    : member.namespace
+    ? `@${member.namespace}`
+    : `${member.id}`;
+}
```

Comment:
None of the callers of this use the namespace option, so it can be removed, and I do not think we want to expose the id.
```suggestion
export function getMemberNameToDisplay(
  member: ProjectMemberResponse,
): string {
  return member.first_name || member.last_name
    ? `${member.first_name} ${member.last_name}`
    : member.namespace
    ? `@${member.namespace}`
    : "(unknown)";
}
```